packages/truffle-db/src/workspace/pouch.ts

```diff
@@ -16,6 +16,13 @@ const resources = {
       { fields: ["sourcePath"] },
     ]
   },
+  compilations: {
+    createIndexes: [
+      { fields: ["compiler"] },
+      { fields: ["contractTypes"]},
+      { fields: ["sources"]}
```

Comment:
We can probably skip these indexes for the time being because they're not primitive types and that makes it harder to reason about what that index actually means.

If we find ourselves looking up Compilation by source or by contract type, we can always add this again later.

Reply:
makes sense. removed!